lldb/source/Plugins/LanguageRuntime/Swift/SwiftLanguageRuntimeNames.cpp

```diff
@@ -419,6 +429,66 @@ CreateRunThroughTaskSwitchingTrampolines(Thread &thread,
   return nullptr;
 }
 
+// Search all modules for `target_func` and creates a RunToAddress plan if a
+// single function is found.
+static ThreadPlanSP CreateRunToAddressPlan(StringRef target_func,
+                                           Thread &thread, bool stop_others) {
+  ModuleList modules = thread.GetProcess()->GetTarget().GetImages();
+  SymbolContextList sc_list;
+  modules.FindFunctionSymbols(ConstString(target_func), eFunctionNameTypeFull,
+                              sc_list);
+  if (sc_list.GetSize() != 1 || sc_list[0].symbol == nullptr)
+    return nullptr;
+  Symbol &thunk_symbol = *sc_list[0].symbol;
+  Address target_address = thunk_symbol.GetAddress();
+  if (target_address.IsValid())
+    return std::make_shared<ThreadPlanRunToAddress>(thread, target_address,
+                                                    stop_others);
+  return nullptr;
+}
+
+/// Demangle `symbol_name` and extracts the text at the node described by
+/// `node_path`, if it exists.
+static std::optional<std::string>
+FindClassName(StringRef symbol_name, llvm::ArrayRef<Node::Kind> node_path) {
+  swift::Demangle::Context ctx;
+  NodePointer demangled_node =
+      SwiftLanguageRuntime::DemangleSymbolAsNode(symbol_name, ctx);
+
+  NodePointer class_node = childAtPath(demangled_node, node_path);
```

Comment:
Even if childAtPath handles null NodePointer arguments, still seem easier to follow if you checked the return here.